travis-acceptance-tests.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# exit on error
+set -e
+
+# create debug build
+echo "[INFO] changing directory to android/source"
+cd android/source
+echo "[INFO] Changed directory to : " 
+pwd
+echo "[INFO] running assembleProdDebug ..."
+./gradlew assembleProdDebug
+echo "[INFO] build created"
+cd ../..
+
+# get current timestamp, to be used in apk name
+current_time=$(date "+%Y.%m.%d-%H.%M.%S")
+echo "Current Time : $current_time"
+
+# enter tests project folder
+cd AcceptanceTests
+pwd
+
+# export required keys and parameters
+export SAUCE_KEY="a24b5fd3-7533-4b5e-b063-aacf75956347"
+export SAUCE_USERNAME="rohan-dhamal-clarice"
+export APK_PATH="../android/source/VideoLocker/build/outputs/apk/VideoLocker-prod-debug.apk"
+export APK_NAME="edx-prod-debug-"$current_time".apk"
```

Comment:
@aleffert These are the 4 variables we export. But, we need to keep safe just two of them i.e. SAUCE_KEY and SAUCE_USERNAME. Please set these up in Travis environment and please use key and username from that SauceLabs account that we want to use.